doc/source/data/faq.rst

```diff
@@ -0,0 +1,336 @@
+parallel and pipelined data processing on Ray, providing a higher-level API while
+internally handling data batching, task parallelism and pipelining, and memory
+management.
+
+Who is using Ray Datasets?
+==========================
+
+To give an idea of Datasets use cases, we list a few notable users running Datasets
+integrations in production below:
+
+* Predibase is using Ray Datasets for ML ingest and batch inference in their OSS
+  declarative ML framework, `Ludwig <https://github.com/ludwig-ai/ludwig>`__, and
+  internally in their `AutoML product <https://predibase.com/>`__.
+* Amazon is using Ray Datasets for large-scale I/O in their scalable data catalog,
+  `DeltaCAT <https://github.com/ray-project/deltacat>`__.
+* Shopify is using Ray Datasets for ML ingest and batch inference in their ML platform,
+  `Merlin <https://shopify.engineering/merlin-shopify-machine-learning-platform>`__.
+* Ray Datasets is used as the data processing engine for the 
+  `Ray-based Apache Beam runner <https://github.com/ray-project/ray_beam_runner>`__.
+* Ray Datasets is used as the preprocessing and batch inference engine for
+  :ref:`Ray AIR <air>`.
+
+
+If you're using Ray Datasets, please let us know about your experience on the
+`Slack <https://forms.gle/9TSdDYUgxYs8SA9e8>`__  or
+`Discourse <https://discuss.ray.io/>`__; we'd love to hear from you!
+
+What should I use Ray Datasets for?
+===================================
+
+Ray Datasets is the standard way to load, process, and exchange data in Ray libraries
+and applications, with a particular emphasis on ease-of-use, performance, and
+scalability in both data size and cluster size. Within that, Datasets is designed for
+two core uses cases:
+
+* **ML (training) ingest:** Loading, preprocessing, and ingesting data into one or more
+  (possibly distributed) model trainers.
+* **Batch inference:** Loading, preprocessing, and performing parallel batch
+  inference on data.
+
+We have designed the Datasets APIs, data model, execution model, and
+integrations with these use cases in mind, and have captured these use cases in
+large-scale nightly tests to ensure that we're hitting our scalability, performance,
+and efficiency marks for these use cases.
+
+See our :ref:`ML preprocessing feature guide <datasets-ml-preprocessing>` for more
+information on this positioning.
+
+What should I not use Ray Datasets for?
+=======================================
+
+Ray Datasets is not meant to be used for generic ETL pipelines (like Spark) or
+scalable data science (like Dask, Modin, or Mars). However, each of these frameworks
+are :ref:`runnable on Ray <data_integrations>`, and Datasts integrates tightly with
+these frameworks, allowing for efficient exchange of distributed data partitions often
+with zero-copy. Check out the
+:ref:`dataset creation feature guide <dataset_from_in_memory_data_distributed>` to learn
+more about these integrations.
+
+Datasets is specifically targeting
```

Comment:
nit: unintended line break?

Reply:
Yeah I was refactoring and didn't feel like rebalancing all lines to ~88 characters. 😛 It doesn't change the rendered text. But I can fix this while making the other changes.